prow/cmd/deck/static/spyglass/spyglass.ts

```diff
@@ -51,7 +51,17 @@ function requestReload(name: string, body: string): void {
 // Insert rendered view into page and remove loading wheel
 function insertView(name: string, content: string): void {
   document.getElementById(name + "-loading")!.style.display = "none";
-  document.getElementById(name + "-view")!.innerHTML = content;
+
+  const view = document.getElementById(name + "-view");
```

Comment:
Put the `!` here so you don't need it everywhere else you use `view`.

```suggestion
  const view = document.getElementById(name + "-view")!;
```